### PR TITLE
Bump scaffold version to latest fulcio release

### PR DESCRIPTION
**Pull request by priyawadhwa**
Signed-off-by: Priya Wadhwa <priya@chainguard.dev>

## Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [ ] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [ ] JSON Schema generated.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: charts/scaffold/Chart.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: Scaffolding the components of the sigstore architecture
 
 type: application
 
-version: 0.3.21
+version: 0.3.22
```
Comment (cpanato) [inline charts/scaffold/Chart.yaml:7]: if there is a breaking change we should bump here as well `0.4.0`

Comment (k4leung4) [reply charts/scaffold/Chart.yaml:7]: so fulcio helm chart 0.5.x was cut for fulcio app v0.6.x

looking at fulcio app v0.6.x, there doesnt seem to be any breaking change.
not sure what the reason fulcio was bumped to 0.6.x? https://github.com/sigstore/fulcio/releases/tag/v0.6.0

Comment (haydentherapper) [reply charts/scaffold/Chart.yaml:7]: "Change username format" was technically a breaking change - However, it's unlikely anyone was using this format, and we weren't using this in production. But it was a change in the format of a subject

Comment (priyawadhwa) [reply charts/scaffold/Chart.yaml:7]: Sounds like it's ok to make this upgrade. Could someone PTAL?

**Review by k4leung4 — APPROVED** [review]

File: charts/scaffold/Chart.yaml (hunk 1 above)
Comment (k4leung4) [reply charts/scaffold/Chart.yaml:7]: approved

Comment (priyawadhwa) [reply charts/scaffold/Chart.yaml:7]: thanks kenny!

**Pull request merged**